DiztinGUIsh/util/Perf.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Diagnostics.Tracing;
+
+namespace DiztinGUIsh.util
+{
+    /*[EventSource(Name = "GridUI")]
+    internal sealed class DizUIGridTrace : EventSource
+    {
+        internal static class DebugCounters
+        {
+            public static int CellValueNeeded;
+            public static int CellPainted;
+            public static int ScrollEvents;
+        }
+
+        private void _IncrementEvent(int eventId, ref int value) =>
+            WriteEvent(eventId, Interlocked.Increment(ref value));
+
+        public void Message(string message) => WriteEvent(1, message);
+        public void CellValueNeeded() => _IncrementEvent(2, ref DebugCounters.CellValueNeeded);
+        public void CellPainted() => _IncrementEvent(3, ref DebugCounters.CellPainted);
+        public void ScrollEvent() => _IncrementEvent(4, ref DebugCounters.ScrollEvents);
+
+        public static readonly DizUIGridTrace Log = new();
+    }*/
+    
+    [EventSource(Name="GridUI")]
+    public sealed class DizUiGridTrace : EventSource {
+
+        public static readonly DizUiGridTrace Log = new();
+
+        public class Tasks {
+            public const EventTask CellValueNeeded = (EventTask)0x1;
+            public const EventTask CellPainting = (EventTask)0x2;
+            public const EventTask SelectCell = (EventTask)0x3;
+        }
+
+        private const int CellValueNeededStart = 1;
+        private const int CellValueNeededStop = 2;
+        private const int CellPaintingStart = 3;
+        private const int CellPaintingStop = 4;
+        private const int SelectCellStart = 5;
+        private const int SelectCellStop = 6;
+
```

Comment:
might want to remove this, we're super-fast now that we're using Datasubset, no need to work so hard.